src/friendly-iframe-embed.js

```diff
@@ -465,6 +489,13 @@ export class FriendlyIframeEmbed {
     } else {
       this.signals_.signal(CommonSignals.RENDER_START);
     }
+
+    // TODO(ccordry): remove when no-signing launched.
```

Comment:
FYI: this class and this mode are used by most of our unit tests. So we'd need to switch them to streaming as well.